src/Plugin/Queue/Handlers/Source/DropSourceHandler.php

```diff
@@ -103,6 +103,12 @@ protected static function removeSourceRowData(array $sourceRow, Client $client):
 		foreach ($queries as $query) {
 			$request = $client->sendRequest($query);
 			if ($request->hasError()) {
+				if (str_contains(
+					(string)$request->getError(),
+					"unknown table '_views' in update request"
```

Comment:
Maybe we should not stick with table name here or use some constant, when we will need to rename it we will get different error (we have system table upcoming)